pdf_id/pdf_id.py

```diff
@@ -542,8 +542,13 @@ def analyze_pdf(self, request, res_txt, path, working_dir, heur, additional_keyw
                                     with open(crvf, 'wb') as f:
                                         f.write(con_bytes)
                                     try:
-                                        request.add_extracted(crvf, os.path.basename(crvf),
-                                                              "Extracted content from object {}" .format(k))
+                                        if keyw in ['URI']:
+                                            # Add as supplementary
+                                            request.add_supplementary(crvf, os.path.basename(crvf),
+                                                                      "Supplementary content from object {}" .format(k))
```

Comment:
why not f string?

Reply:
Probably a holdover from v3 code, can go through and update them